components/brave_ads/core/internal/account/wallet/wallet_info.cc

```diff
@@ -11,12 +11,16 @@ bool WalletInfo::IsValid() const {
   return !payment_id.empty() && !public_key.empty() && !secret_key.empty();
 }
 
-bool WalletInfo::WasUpdated(const WalletInfo& other) const {
-  return *this != other;
+bool WalletInfo::WasUpdated(const WalletInfo& previous) const {
+  return *this != previous;
 }
 
-bool WalletInfo::HasChanged(const WalletInfo& other) const {
-  return other.IsValid() && WasUpdated(other);
+bool WalletInfo::HasChanged(const WalletInfo& previous) const {
+  return previous.IsValid() && WasUpdated(previous);
+}
+
+bool WalletInfo::WasUpdatedFromScratch(const WalletInfo& previous) const {
```

Comment:
Maybe `WalletInfo::WasCreated`

Reply:
Hmmm but the wallet was not created, just not sure about `FromScratch` :-)

Reply:
`WalletInfo::WasCreated` is a better naming here. Thanks!